suite-native/atoms/src/Link.tsx

```diff
@@ -0,0 +1,13 @@
+import React, { useCallback } from 'react';
+import {Linking, TouchableOpacity, TouchableOpacityProps} from 'react-native';
+
+type LinkProps = TouchableOpacityProps & {
+    url: string;
+};
+
+export const Link = ({ url, ...touchableProps }: LinkProps) => {
+    const handleRedirect = useCallback(async () => {
+        if (await Linking.canOpenURL(url)) Linking.openURL(url);
```

Comment:
Maybe add some toast message in case of fail of this.

Reply:
fixed